### PR TITLE
Fix Django 1.9/1.10 deprecation notices.

### PR DESCRIPTION
**Pull request by Osmose**
Fixes some issues with features going away in Django 1.10. As a result, this also bumps us up to Django 1.8 as the minimum Django version supported.

This depends on #54. @pmclanahan r?

### CODE REVIEW

**Comment by pmclanahan** [comment]
#54 is merged. Looks like maybe this needs a rebase now?

**Comment by Osmose** [comment]
Rebased!

**Comment by pmclanahan** [comment]
@Osmose Travis is complaining I think because the .travis.yml file needs to be updated to coincide with the env changes in tox.

File: tox.ini [hunk 1]
```diff
@@ -1,14 +1,13 @@
 [tox]
 args_are_paths = false
 skip_missing_interpreters = true
-envlist = py{27,33,34,py}-dj{17,18},py{27,34,35}-dj19,py35-dj18,flake8
+envlist = py{27,33,34,py}-dj18,py{27,34,35}-dj19,py35-dj18,flake8
```
Comment (pmclanahan) [inline tox.ini]: Please add "35" to the first list and remove "py35-dj18".

**Comment by pmclanahan** [comment]
A couple of tweaks for travis and tox then I think we're ready. Thanks!

PS - I use the following to generate the list for the `.travis.yml` file when the `tox.ini` changes:

```bash
$ for env in $(tox -l); do echo "  - TOXENV=${env}"; done
```

**Comment by Osmose** [comment]
Updated, travis is happy now. :D

**Comment by pmclanahan** [comment]
w00t :dancers:

**Pull request merged**